pkg/agent/mocks/httpsmock.go

```diff
@@ -0,0 +1,31 @@
+package mocks
+
+import (
+	"fmt"
+	"github.com/kubeflow/kfserving/pkg/agent/storage"
+	"net/http"
+	"net/url"
+)
+
+type mockHTTPSClient struct {
```

Comment:
I think you can use the `"net/http/httptest"` package

Reply:
I'll check it out and see if I can use that package instead. Thanks!